ivy_tests/test_ivy/helpers.py

```diff
@@ -2367,6 +2376,15 @@ def get_axis(
                 | st.just(0)
                 | st.lists(st.just(0), min_size=min_size, max_size=max_size)
             )
+        elif just_tuple = True:
```

Comment:
No need of this if block, instead at the last line of the function where we return axis, just add an if block to check if `ret_tuple` is `True` and then return `tuple(axis)`.